From 0cc779eb7eb2d10475e38714fca663c51c46ec9d Mon Sep 17 00:00:00 2001
From: Yifeng Zhao <zyf@rock-chips.com>
Date: Tue, 8 May 2018 09:26:14 +0800
Subject: [PATCH] soc: rockchip: vendor storage: include rk_vendor_storage.h
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fix the following warning(s):
drivers/soc/rockchip/rk_vendor_storage.c:15:5: warning: no previous prototype for ‘rk_vendor_read’ [-Wmissing-prototypes]
drivers/soc/rockchip/rk_vendor_storage.c:23:5: warning: no previous prototype for ‘rk_vendor_write’ [-Wmissing-prototypes]
drivers/soc/rockchip/rk_vendor_storage.c:31:5: warning: no previous prototype for ‘rk_vendor_register’ [-Wmissing-prototypes]
drivers/soc/rockchip/rk_vendor_storage.c:42:6: warning: no previous prototype for ‘is_rk_vendor_ready’ [-Wmissing-prototype

Change-Id: Ia264c96adac27ff1e639532b7bdfde464f54e637
Signed-off-by: Yifeng Zhao <zyf@rock-chips.com>
---
 drivers/soc/rockchip/rk_vendor_storage.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/soc/rockchip/rk_vendor_storage.c b/drivers/soc/rockchip/rk_vendor_storage.c
index 0ff5a09b4fbf..0d1dc5241a40 100644
--- a/drivers/soc/rockchip/rk_vendor_storage.c
+++ b/drivers/soc/rockchip/rk_vendor_storage.c
@@ -8,6 +8,7 @@
  */
 
 #include <linux/kernel.h>
+#include <linux/soc/rockchip/rk_vendor_storage.h>
 
 int (*_vendor_read)(u32 id, void *pbuf, u32 size) = NULL;
 int (*_vendor_write)(u32 id, void *pbuf, u32 size) = NULL;
-- 
2.35.3

